CRM/Case/BAO/Case.php

```diff
@@ -1151,9 +1151,18 @@ public static function getCaseActivity($caseID, &$params, $contactID, $context =
 
       // Activity Status Label for Case activities list
       $caseActivities[$caseActivityId]['status_id'] = CRM_Core_PseudoConstant::getLabel('CRM_Activity_BAO_Activity', 'activity_status_id', $dao->status);
+      $deleted = '';
+      if ($dao->deleted) {
+        $deleted = '<br /> (deleted)';
```

Comment:
I know it's not part of the refactor, but just noting that the string "deleted" should get passed through `ts()`

Reply:
@colemanw Good spot - makes sense to fix it now though, and I've fixed another one too.